nest/03.nest.md

```diff
@@ -0,0 +1,89 @@
+# 2 nest
+In this chapter, we will detail how to use the binary `nest`.
+As explained before, this binary is mainly designed for ease of use.
+However, anyone can use it.
+
+## 2.1 nest options
+`nest` has three options available:
+
+* `-h` or `--help`, which prints the help information of `nest`, consisting of a short explanation for each of `nest`'s options and commands.
+* `-v`, which sets the level of verbosity for `nest`. Basically, how much `nest` will explain what it's doing in the terminal.
+* `-V` or `--version`, which prints the version of `nest` you are currently using.
+
+To use them, simply type `nest` followed by any option. For example, `nest -h`.
+
+## 2.2 nest commands
+### 2.2.1 Updating the list of available packages with `nest pull`
+This command synchronizes the cache of local packages with the one on Raven's server.
```

Comment:
Instead of starting by saying what the command does and then why it is useful, maybe explain first the problematic and then the solution.

Something like "Nest needs the list of available packages to know what to do when an install is requested ... you need to do that often for that and that reason ... if you don't horrible things might happen like ... `nest pull` is used for that purpose ..."

What do you think?

Reply:
I'd let the first sentence define what it is and afterwards add a rational. This allows user that already know the "problem" to skip it.

Reply:
that makes sense, I guess it should be that way for the other commands too
I'm not sure to understand what you meant by 'rational' @Gaspard--

Reply:
>Something like "Nest needs the list of available packages to know what to do when an install is requested ... you need to do that often for that and that reason ... if you don't horrible things might happen like ... nest pull is used for that purpose ..."

This is a rational, it explains why the command is needed.

Reply:
Okay, why not =)